src/orga/minutes/2022-12-14.md

```diff
@@ -0,0 +1,99 @@
+# conda-forge core meeting 2022-12-14
+
+Add new agenda items under the `Your __new__() agenda items` heading
+
+[last weeks meeting](https://hackmd.io/jEpu4WolRv294NzZ3AJdOw)
+[What time is the meeting in my time zone](https://arewemeetingyet.com/UTC/2020-08-26/17:00/w/Conda-forge%20dev%20meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9wUk15dFVKV1FmU3NJM2xvMGlqQzJRP2VkaXQifQ==)
+Meeting info: 
+* To join the video meeting, click this link: https://zoom.us/j/9138593505?pwd=SWh3dE1IK05LV01Qa0FJZ1ZpMzJLZz09
+* Otherwise, to join by phone, dial +1 347-384-8597 and enter this PIN: 828 997 153#
+* To view more phone numbers, click this link: https://tel.meet/ijv-qsvm-tvn?hs=5
+
+## Attendees
+
+| Name                    | Initials | GitHub ID        | Affiliation                 |
+| ----------------------- | -------- | ---------------  | --------------------------- |
+| Dave Clements           | DPC      | tnabtaf          | Anaconda                    |
+| Marcel Bargull          | MB       | mbargull         | Bioconda/cf                 |
+| Cheng H. Lee            | CHL      | chenghlee        | Anaconda/cf                 |
+| Matthew Becker          | MRB      |  beckermr        | cf                          |
+| Marius van Niekerk      | MvN      | mariusvniekerk   | cf                          |
+| John Kirkham            | JK       | jakirkham        | NVIDIA/cf                   |
+| Jaime Rodríguez-Guerra  | JRG      | jaimergp         | Quansight/cf                |
+|                         |          |                  |                             |
+|                         |          |                  |                             |
+
+10 people total
+
+
+### Standing items
+
+* [ ] intros for new folks on the call
+
+* [ ] open votes
+
+### From previous meeting(s)
+
+* [ ] (JL) Upcoming conda 22.11.0, conda-libmamba-solver releases
+    * [ ] https://github.com/conda/conda/releases/tag/22.11.0
+        * [ ] NEW! Download and extract packages in parallel
+        * [ ] NEW! plugins (launching with solver, subcommands and virtual packages plugin hooks)
+            * [ ] E.g. cuda virtual package: https://github.com/conda/conda/blob/main/conda/plugins/virtual_packages/cuda.py
+        * [ ] Switched to ruamel.yaml (look ma, no underscore!)
```

Comment:
i think i'm missing the reference here. What is the joke?

Reply:
We relied on a fork that was called `ruamel_yaml` for way too long and we finally dropped it. It was always a source of confusion.